### DISCUSSION TITLE
Qt library executable path

### DISCUSSION THREAD

**Discussion by gogochikago** [qutebrowser/qutebrowser#5570] [Q&A] [answered]
qutebrowser v1.13.0
Git commit: 
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.0

CPython: 3.8.3
PyQt: 5.15.0

sip: 5.3.0
colorama: no
pypeg2: 2.15
jinja2: 2.11.2
pygments: 2.6.1
yaml: 5.3.1
cssutils: no
attr: 19.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.0
PyQt5.QtWebKitWidgets: no
pdf.js: 2.3.200 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.31.1
QtNetwork SSL: OpenSSL 1.1.1d  10 Sep 2019

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel Open Source Technology Center, 3.0 Mesa 19.3.4
Platform: Linux-5.5.7-arch1-1-x86_64-with-glibc2.29, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /home/stas/.asdf/installs/python/3.8.3/lib/python3.8/site-packages/qutebrowser
Using Python from /home/stas/.asdf/installs/python/3.8.3/bin/python3.8
Qt library executable path: /home/stas/.asdf/installs/python/3.8.3/lib/python3.8/site-packages/PyQt5/Qt/libexec, data path: /home/stas/.asdf/installs/python/3.8.3/lib/python3.8/site-packages/PyQt5/Qt

Paths:
cache: /home/stas/.cache/qutebrowser
config: /home/stas/.config/qutebrowser
data: /home/stas/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:00:02

How can i manualy change Qt library executable path? Cuz some videos can't play

**Comment by The-Compiler** [comment, marked as answer]
qutebrowser uses whatever PyQt/Qt you installed with the Python you're running it. Here, it looks like you're running qutebrowser with `/home/stas/.asdf/installs/python/3.8.3/bin/python3.8` which has a `PyQt5` installed via `pip install` in `/home/stas/.asdf/installs/python/3.8.3/lib/python3.8/site-packages/PyQt5`.

You might want to instead install qutebrowser via `pacman` (given that you're on Arch), or perhaps uninstall the custom PyQt via something like `/home/stas/.asdf/installs/python/3.8.3/bin/python3.8 -m pip uninstall PyQt5`.

**Reply by gogochikago** [reply]
Thanks! It works.
